unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/WorldChatWindowHUD/PublicChatWindowController.cs

```diff
@@ -27,7 +31,12 @@ private enum ChatWindowVisualState { NONE_VISIBLE, INPUT_MODE, PREVIEW_MODE }
     private CancellationTokenSource deactivateFadeOutCancellationToken = new CancellationTokenSource();
 
     private bool skipChatInputTrigger;
-    
+    private string lastPrivateMessageRecipient = string.Empty;
+
+    private UserProfile ownProfile => userProfileBridge.GetOwn();
+    internal BaseVariable<HashSet<string>> visibleTaskbarPanels => DataStore.i.HUDs.visibleTaskbarPanels;
```

Comment:
please avoid coupling singleton inside controllers, use the `dataStore` field that is already set